internal/fositestorage/authorizationcode/authorizationcode.go

```diff
@@ -231,55 +232,19 @@ const ExpectedAuthorizeCodeSessionJSONFromFuzzing = `{
 						"kty": "OKP",
 						"crv": "Ed25519",
 						"x": "LHMZ29A64WecPQSLotS8hfZ2mae0SR17CtPdnMDP7ZI",
-						"x5u": {
-							"Scheme": "",
-							"Opaque": "",
-							"User": null,
-							"Host": "",
-							"Path": "",
-							"RawPath": "",
-							"OmitHost": false,
-							"ForceQuery": false,
-							"RawQuery": "",
-							"Fragment": "",
-							"RawFragment": ""
-						}
+						"x5u": "https://x5u.example.com"
```

Comment:
@benjaminapetersen FYI, I could tell that there was a change in the format of the stored session objects because the test that uses this expected value failed, and I needed to update the expected value to make it pass.

Reply:
👍